benchmarks/smallmatvec/manual_versions/openmp/utils.F90

```diff
@@ -0,0 +1,45 @@
+module utils
+    implicit none
+    contains
+subroutine system_mem_usage(valueRSS)
+    use ifport !if on intel compiler
```

Comment:
Please add a brief docstring, mentioning any limitations. (e.g. just ifort? and presumably only certain flavours of OS?_

Reply:
Done

Reply:
Done